### PR TITLE
Added support dnos9_facts module

### PR DESCRIPTION
**Pull request by skg-net**
##### ISSUE TYPE
- New Module Pull Request
##### COMPONENT NAME
- modules/core/network/dnos9/dnos9_facts
##### ANSIBLE VERSION
- ansible 2.2.0 (devel a194740) last updated 2016/09/08 17:44:28 (GMT -700) lib/ansible/modules/core: (detached HEAD db38f0c876) last updated 2016/09/08 17:44:58 (GMT -700) lib/ansible/modules/extras: (detached HEAD 8bfdcfcab2) last updated 2016/09/08 17:44:58 (GMT -700) config file = configured module search path = Default w/o overrides 
##### SUMMARY
- Added support OS9 facts module to support Dell Networking OS9 devices.

### CODE REVIEW

**Comment by skg-net** [comment]
@privateip @gundalow, please have look at it

File: network/dnos9/dnos9_facts.py [hunk 1]
```diff
@@ -0,0 +1,553 @@
...
+  returned: always
+  type: string
+ansible_net_image:
+  description: The image file the device is running
+  returned: always
+  type: string
+
+# hardware
+ansible_net_filesystems:
+  description: All file system names availabe on the device
```
Comment (gundalow) [inline network/dnos9/dnos9_facts.py]: available

**Review by gundalow — CHANGES_REQUESTED** [review]
needs_revision

I notice that this is by @dhivyap, should I add @dhivyap as a maintainer for dnos*, just so it's not only you that get notified when other people start raising issues/PR

**Comment by gregdek** [comment]
Thanks again to @skg-net. This module is going into community review.

We notice that you are one of the maintainers (@dhivyap, @skg-net, @privateip, @gundalow) of this module, so when you think it's ready, please comment "shipit" and we will consider it for merging.

[This message brought to you by your friendly Ansibull-bot.]

File: network/dnos9/dnos9_facts.py (hunk 1 above)
Comment (skg-net) [reply network/dnos9/dnos9_facts.py]: Done. Please add @dhivyap as well.
ready_for_review

**Comment by skg-net** [comment]
ready_for_review

**Comment by gundalow** [comment]
Merged, thanks

**Pull request merged**